s3parq/publish_parq.py

```diff
@@ -91,6 +87,8 @@ def _assign_partition_meta(bucket: str, key: str, dataframe: pd.DataFrame, parti
                 head_obj = s3_client.head_object(Bucket=bucket, Key=obj['Key'])
                 if not 'partition_data_types' in head_obj['Metadata']:
                     all_files_without_meta.append(obj['Key'])
+                if redshift_params:
```

Comment:
Is these a need for this if statement? The function signature does not allow this to not be sent (does not default to None)